src/examples/napi_sine/README.md

```diff
@@ -1,5 +1,12 @@
 ﻿# C++ example using Network API
-The program `hotgym_napi` is an example of an app using the Network API tools available in the htm.core library.  In this example we generate a sin wave with some noise as the input.  This is passed to an encoder to turn that into SDR format.  This is passed to two instances of SpatialPooler (SP), one is configured for local inhibition and one for global inhibition.  The output of the SP for global inhibition is passed on to the temporalMemory (TM) algorithm.  The output of the TM can be written to a file so that it can be plotted.
+The program `sine_napi` is an example of an app using the Network API tools 
+available in the htm.core library.  In this example we generate a sine wave 
+with some noise as the input.  This is passed to an encoder to turn that 
+into SDR format.  This is passed to two instances of SpatialPooler (SP), 
```

Comment:
nit, SP makes the SDR format, encoder does "just" binary representation of whatever raw inputs. Also, do you want to mention the 2 SP instances, when only one is actually used?

Reply:
> do you want to mention the 2 SP instances

Oh, yes. I should do that.  Actually, I think I will take out the second SP entirely.  This is kind of a Hello World example and should be as simple as possible.